docs/polkadotjs_apps_public.md

```diff
@@ -1,32 +1,34 @@
 ---
 id: polkadotjs_apps_public 
-title: Connect to public nodes
+title: Connect to a public node
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-This are public RPC nodes running by hydraDX and our partners. You can use this nodes to interact with snakenet. To connect you one of this nodes just click on one of links bellow.
+There are two public RPC nodes which are maintaned by HydraDX and our partners. You can use these nodes for interacting with Snakenet. You can directly connect to a public node with the Polkadot.JS app by clicking on the links below:
```

Comment:
Wording in the first sentence is still little bit off. Also naming of Polkadot/apps

Reply:
Still not resolved